### ISSUE TITLE
Map visualizer - list UX

### ISSUE THREAD

**Issue by jirihelmich** [tobice/payola-viz#7]
- [x] Enable the user to quickly toggle the state of an item (concept instance) by clicking the leftmost icon. It will rotate the states always -> never -> defined. It's much quicker than using the overflow menu for every single item.

**Comment by jirihelmich** [comment]
- [x] Add the possibility to enlarge the window, it's very small and hard to work with, when lists are larger.

**Comment by jirihelmich** [comment]
- [x] I don't see how many concepts are somehow active. Some summary in the header would be great.

**Comment by tobice** [comment]
>  Enable the user to quickly toggle the state of an item (concept instance) by clicking the leftmost icon. It will rotate the states always -> never -> defined. It's much quicker than using the overflow menu for every single item.

Implemented in 1e586f3.

> Add the possibility to enlarge the window, it's very small and hard to work with, when lists are larger.

Won't fix for now. I don't really see how enlarging the window would solve the problem. I created a separate issue #16.

>  I don't see how many concepts are somehow active. Some summary in the header would be great.

Implemented in 99733f0. Note that the number of active concepts is displayed only in the preview mode/published application. In the configuration mode, the situation is more complex (three different modes for each option/concept), so at this moment, nothing is displayed.

Note that for large filters (~100 concepts) the UX will always be cumbersome (I'm not saying it cannot be improved). The point of the application generator is that the user can decide whether he wants to generate an application with all filters/options or with just a subset, making the UX significantly better for the end user.
